.gitignore

```diff
@@ -3,8 +3,8 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` directory is correctly ignored, but ensure that this is the correct virtual environment directory name used in your project. If your virtual environment is named differently, you should update this entry accordingly.